### DISCUSSION TITLE
Invalid domain configuration, but looks OK

### DISCUSSION THREAD

**Discussion by shawnsrk** [vercel/community#398] [Help] [answered]
Today, Vercel started "complaining" about an invalid domain configuration for my website.
![screenshot](https://user-images.githubusercontent.com/8838364/156651158-6bd568cd-0d66-47b0-9448-5bea02325317.png)

I am not an expert on DNS records but, as best as I can tell, the configuration is OK
- It's asking me to add an A record that's already there
- It's asking me to add CNAME records that point to cname.vercel-dns.com, but there is already a * record that points there.

The issue seems almost identical to this [one](https://github.com/vercel/vercel/discussions/5269). We also just had our domain renewed.

I am sorry to ask a support style question here, but I am at my wits on how to solve the problem.

I am already submitted the support ticket but I am worried about the long response time.

**Comment by JTaylor0196** [comment]
Hey there, @shawnsrk!

Thanks for reaching out about this. The team is aware of the issue and is currently looking into it. I'll circle back around with an update once I have some more information!

**Comment by shawnsrk** [comment, marked as answer]
Vercel Support resolved the issue. Per my email discussion with them, this issue was caused by something on Vercel's end, not by bad configuration on my part.

**Reply by Pieparker** [reply]
Hey there! Just to follow up – this was associated with a sporadic, short-lived issue with an upstream provider, upsetting a handful of DNS records within the UI. This was promptly addressed.

If you have a similar issue in future, please do create a discussion and we'd be happy to dig in!
